cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -127,6 +128,8 @@ func addHostFolder(cmd *cobra.Command, args []string) {
 
 func addSSHFSInteractive(manager *hostFolderConfig.Manager, name string) error {
 	source := util.ReadInputFromStdin("Source path")
+	source, err := homedir.Expand(source)
+	fmt.Println(source, err)
```

Comment:
I think this line only for test purpose and also you need to handle this err, mostly return if not `nil`.

Reply:
Yes..i was testing.. sorry forgot to put the WIP